### PR TITLE
Adds .38 ANCR Rounds (Bluespace grounding rounds) (blatant recreation of #84668)

### PR DESCRIPTION
**Pull request by Stalkeros2**
## About The Pull Request
~~You know I've already copied the code so might as well do the same for text; screenshots are all-mine however.~~
image
image
image
Adds the ANCR rounds (it's short for "anchor"), a low-damage 38 caliber round that implants a temporary bluespace grounding implant, much like its sister ammo, the TRAC round.
ANCR rounds are available under the Exotic Ammo techweb node, like Hotshot/Iceblox rounds.
The lifespan is only a minute, which is forever in combat time, but completely shutting down escapes is more powerful than simply tracking where they went.
## Why It's Good For The Game
~~itired chasing teleporting blorbos :(~~
Adds a late-round solution for slippery teleport-spamming antags, Further aligns the detective as a support role for security like how TRAC rounds enable similar behavior, but in a more direct way.
## Changelog
:cl: Stalkeros, YakumoChen for original PR
add: ANCR .38 rounds, combat implant technode. .38 bullets that implant a short-lived bluespace grounding implant.
/:cl:

### CODE REVIEW

**Comment by YakumoChen** [comment]
Oh thanks

**Comment by YakumoChen** [comment]
Now I remember why I closed this, it still needs a status effect coded in to make it more clear to the victim

File: code/game/objects/items/implants/security/implant_noteleport.dm [hunk 1]
```diff
@@ -56,3 +56,24 @@
 	name = "implant case - 'Bluespace Grounding'"
 	desc = "A glass case containing a bluespace grounding implant."
 	imp_type = /obj/item/implant/teleport_blocker
+
+/obj/item/implant/teleport_blocker/c38
+	name = "ANCR implant"
+	desc = "A smaller bluespace grounding implant that supplies power for only a few minutes."
```
Comment (YakumoChen) [inline code/game/objects/items/implants/security/implant_noteleport.dm:62]:
```suggestion
	desc = "A smaller bluespace grounding implant that supplies power for only one minute."
```

Comment (Stalkeros2) [reply code/game/objects/items/implants/security/implant_noteleport.dm:62]: i forgot 💀

**Comment by EnterTheJake** [comment]
This is balls to the wall overpowered wtf.

The grounding implant was designed as a tool to contain prisoners, It waa never meant to be a combat debuff, for good reason.

**Comment by Livrah** [comment]
I like that it's a .38, which is limited to one copy at the start of the round (or 2k from cargo).
Exotic ammo research is a late game, so it won't be an easy counter for wizard, warops or cult.

**Comment by YuliceTheDruid** [comment]
> First of all, this isn't late tech by any stretch of imagination, combat implants can be rushed in 5 minutes, they are not locked behind any type of shell or experiment.

This dude forgot that Sci needs points to research things and that if you disregard the basics then sci will be blown up.

**Comment by Holinka4ever** [comment]
> This is balls to the wall overpowered. You are essentially rendering all antagonists who rely on jaunts useless.

I mean, first of all we need atleast to get our target, and due of lag, time dilation and basic movement, it cant be so easy done.

> First of all, this isn't late tech by any stretch of imagination, combat implants can be rushed in 5 minutes, they are not locked behind any type of shell or experiment.

Except that you need to deal with whole sci departament, that has a lot of stuff to research, and "rushing" it will end by kickin out that sec arm rusher.

> The grounding implant was designed as a tool to contain prisoners, It was never meant to be a combat debuff, for good reason.

These is cool argument, i like it.

> Antagonists have escape tools because they are always outnumbered, them escaping it's fine, they don't need to be ground into paste the very moment they decide to engage on Security.

Maybe its cool, but not gonna lie, heretic that teleporting around station every time that got spot from sec is kinda weird.

> Security is a department, their strenght is in their numbers, if an antagonist is teleporting around, they should be chased.

But if they teleporting, a lot, even if the whole station gonna chase em, they still blinking to the bitter end to give a frustration to anyone on shift.

> This is just an instant "I win" button, it's not interesting nor fun for both parties involved, and makes for an incredibly boring round to be observed.

We have baton that stuns in one hit and lasergun, that gives 3rd rate burns in one shot, the most broken, but also frustrating to fight with and against.

**Comment by EnterTheJake** [comment]
@YuliceTheDruid @Holinka4ever 

You are grossly overstimating the amount of points it takes to get these rounds, it's literally 4 nodes, the HoS can easily queue them from the comfort of their PDA, you don't need to interact with sci at all, they are just that easy to get.

A teleporting antagonist might be annoyng, but picture this, getting caught equals to round removal.

**Comment by moocowswag** [comment]
Same counter to mechs genetics chemistry botany etc, sabotage the department if youre an antagonist and that department can make things that counter your strat....

**Comment by Dorsisdwarf** [comment]
Thats a pretty devastatingly powerful ability to be so shallow on a cheap node that isnt gated behind weapons tech isnt it? Especially since it's a "no warning" ability because its just an ammotype on a weapon that usually does something different

**Comment by EnterTheJake** [comment]
If this is ever going to be considered for a merge, it needs to be dropped to something like 5 seconds and move farther into the research tree.

60 Seconds with no ability to be removed is just nuts.

**Comment by 0ri-0ri** [comment]
1 minute is probably too much. Something within the 1~10 second range might be less overwhelming?

**Comment by the-orange-cow** [comment]
Does this affect heretic blade shatter?  Or just jaunt type abilities like ashen passage?

**Comment by EnterTheJake** [comment]
@the-orange-cow yes it affects blade break.

**Comment by arnociiroc** [comment]
this would be cooler if it relied on embedding instead of a flat application

**Comment by Exester509** [comment]
This is a one-hit kill for hit and run antagonists if you land a single round, especially with the new .38 gun proposal in the form of that one carbine from necromanceranne, that's still in the works IIRC.
Heretics, wizards, nightmares, will lose their main way to avoid immediately getting overwhelmed in a split second.

**Comment by necromanceranne** [comment]
You know, after really thinking about it, while I was initially onboard with ANCR, after testing TRAC as an early round with my BR-38 PR, I think these kinds of hard counter effects are a serious issue with regards to powercreep/arms race between antagonist solutions and crew solutions to problems that is largely worse for antagonists than crew.

TRAC is a comparatively soft counter to what ANCR outright prevents;, it gives opportunities to keep pursuit, but it doesn't stop the antagonist from attempting to give themselves space. I think that is preferable, but even that seemed quite oppressive when I moved it earlier into the round.

If TRAC really started to oppress antagonistic options when it was present in a round at all times, and was only a _soft,_ counter to these tools, ANCR by comparison seems like a no-brainer method to game end a significant portion of potential threats whose survival is entirely reliant on their teleportation abilities, or anyone who is being chased and needs to make getaways using teleports they sourced. (nightmares, wizards, ninjas, heretics (particularly cosmic and ash heretics), some traitor tools, cultists, and anyone who nabbed the handtele or some bluespace objects).

HUGE list of foes shit out of luck. Since sec already has huge damage sticks to take someone out if they CAN'T utilize these tools, and the antagonist is built expecting them to be able to use these to keep them alive else they just die, its a pretty mindless solution to the problem of how to fight them, since a direct conflict with sec is usually a losing battle. While currently .38 is only in the detective's revolver, and that is a little hard to distribute to more than just the detective, there is a future where .38 could be more available (I'm going to re-PR the BR-38, so it isn't out of the question).

**Comment by YakumoChen** [comment]
It might be worth it to make the rounds punish teleporting instead of outright denial. 

I never re-opened the PR since it was hard to really justify it fully in the face of criticism (and because I can't code a status effect)

Some alternatives to this might be introducing a short delay to all teleports (1-2 seconds) or more direct punishment like damage or debilitating statuses to teleporting

**Comment by Loafin34** [comment]
i think having it be a literal embed applying the effect is way cooler. Then, an antag could use a hemostat on their body to remove it in 2 seconds, which is a while in terms of combat.

**Comment by EnterTheJake** [comment]
> i think having it be a literal embed applying the effect is way cooler. Then, an antag could use a hemostat on their body to remove it in 2 seconds, which is a while in terms of combat.

It wouldn't actually make a difference,  jaunts usually require split second decisione, if you gotta stay still chances are you are already dead, same principale for bolas.

**Comment by necromanceranne** [comment]
I would think it better to just not introduce something like this in the first place.

**Comment by Sakamoto4ka** [comment]
newgen antags so weak. I think someone is unhappy that some antag dropped secies 100 winstreak

**Comment by moocowswag** [comment]
> I have no idea why Livrah thought this was preferable to making bluespace crystals less functional. That's nonsense, this is a much more severe response to teleport proliferation that we shouldn't be taking except as a last resort. I much prefer a softer solution than starting to introduce hard counters


Ss13 is a game of hard counters, that being said i just realized it was a bullet that literally implants you instead of embedding which means even if the target briefly gets away they cant take it out….thats really punishing for a ranged projectile thats spammable and crew accessible

**Comment by Loafin34** [comment]
i think its good because it requires the one gun to do it. if you come up to the detective and he's prepared, you better bladebreak in time.

**Comment by EnterTheJake** [comment]
> i think its good because it requires the one gun to do it. if you come up to the detective and he's prepared, you better bladebreak in time.

Revolvers can be ordered and 38. Autoriale are coming in the near future.

**Comment by Loafin34** [comment]
you need to A. get the gun B. get the rounds from exotic ammo and C. be able to shoot at the heretic/traitor. if this was given an especially interesting bullet (like glowing blue or something) it'd be great. Plus, the autorifles are really weird as is. they had 20 rounds for no apparent reason, just bringing back the wt-550.

**Comment by EnterTheJake** [comment]
> you need to A. get the gun B. get the rounds from exotic ammo and C. be able to shoot at the heretic/traitor. if this was given an especially interesting bullet (like glowing blue or something) it'd be great. Plus, the autorifles are really weird as is. they had 20 rounds for no apparent reason, just bringing back the wt-550.

1) it's a roundstart gun.

2) it's in the combat implants node, which can be acquired in 5 minutes worth of points.

3) you only need to hit them once, a speedloadwr has 6 rounds and you can carry multiple.

**Comment by optimumtact** [comment]
make it embed instead of implant and it seems like a reasonable change to counter antags who over rely on teleportation.

If it's considered to be too OP with one shot, you could make it a sliding scale, first shot reduces teleport range, second shot reduces range further, third shot prevents teleport etc.

**Comment by carlarctg** [comment]
a stacking pile of soft counters vs soft strats would be really, really fun for gameplay, if security didn't have the strongest weapon in the game that nullifies almost every item antags use (batong)


because it does have one massively powerful tool, everything needs to be balanced around it which means everything else needs to be much, much weaker than basically any 'normal' iteration of the concept, which includes the notion of anti-teleport ammunition. things like this could be healthy for the game if 1. they werent full counters and 2. sec's entire kit was roughly even in terms of power rather than having 90% of it concentrated on one stick

there is also the fact that our security culture is, well, it's fucking awful. people play to win and not to have fun so antags are never given second chances - rarely even borged. this means that losing as an antag is even worse than it 'should' be


also? i honestly kind of dislike .38 special ammo as a whole, besides trac which is thematically appropiate? it just makes detective even more sweatsec. maybe just make a sec crossobw or something that can fit special bolts

**Comment by Stalkeros2** [comment]
> make it embed instead of implant and it seems like a reasonable change to counter antags who over rely on teleportation. (Written and rephrased by three or so different people.)

~~*In hindsight, I probably should've communicated this earlier.*~~
Yeah that's the plan looking at all the feedback; I'm waiting until I have a bit more free time to figure out how.

**Comment by EnterTheJake** [comment]
> > make it embed instead of implant and it seems like a reasonable change to counter antags who over rely on teleportation. (Written and rephrased by three or so different people.)
> 
> ~_In hindsight, I probably should've communicated this earlier._~ Yeah that's the plan looking at all the feedback; I'm waiting until I have a bit more free time to figure out how.

Also make it so it doesn't affect blade break, it's meant to be a heretic last resort.

**Comment by ju45he** [comment]
If anyone gets this, it should NOT be the detective. The role designed to gather intelligence and do some scouting should NOT hold the hard counter.
TRAC fits the detective's role perfectly, allowing them to find where the enemy is going and be able to coordinate security against them.
ANCR just encourages the detective to be the very first responder and to immediately magdump the enemy to stop them from teleporting away.

**Comment by YakumoChen** [comment]
Magdumping with a 10 force bullet will just get them killed like with TRAC
The Det is a support role, they should be sticking with other security or staying on the peripheral of combat

**Comment by ju45he** [comment]
You fill the first 1 or 2 bullets with ANCR and the rest with normal, you should really never fully load the niche bullets into the gun unless you plan on hitting 1 or 2 of the shots.

**Pull request closed without merging**